resources/schema.json

```diff
@@ -91,6 +91,18 @@
                 }
             }
         },
+        "ignoreMsiWithNoMutations": {
+            "type": "boolean",
+            "description": "Ignore MSI violations with zero mutations."
+        },
+        "minMsi": {
+            "type": "number",
```

Comment:
I think you can add to both:

```json
{
    "minimum": 0,
    "maximum": 100
}
```

Wonder what happens in both extremes, but that's unrelated to this PR

Reply:
👍